### ISSUE TITLE
suggested improvements for default states of detection view in interactive interface

### ISSUE THREAD

**Issue by ShaiberAlon** [merenlab/anvio#805]
I think the following defaults would be useful:

1. for detection view, the default should be no normalization (currently it is log) 
2. for detection view - max automatically set to 1

**Comment by meren** [comment]
@ozcan is refactoring large parts of the code, and he may be able to address this as a part of that effort. 

@ozcan, we don't have defaults for different views. Do you think we can deliver this?

**Comment by meren** [comment]
Well, thinking just a bit more about this I realize creates another design problem :(

Because if we want to do this, we will have to set these defaults only for layers that are our 'samples', but the interface doesn't know which layer is a 'sample' and which layer is an additional layer, etc.

**Comment by ozcan** [comment]
Good point @meren, we know sample names in interactive so we can let interface know too.

**Comment by meren** [comment]
This is still not done, but I think we should certainly address.
